tools/cosmovisor/args.go

```diff
@@ -28,6 +28,7 @@ const (
 	EnvDataBackupPath       = "DAEMON_DATA_BACKUP_DIR"
 	EnvInterval             = "DAEMON_POLL_INTERVAL"
 	EnvPreupgradeMaxRetries = "DAEMON_PREUPGRADE_MAX_RETRIES"
+	EnvDisableLogs          = "DAEMON_DISABLE_LOGS"
```

Comment:
The name may be confusing, as it disables cosmovisor logs and not the app logs.
Something like `COSMOVISOR_DISABLE_LOGS` would be a bit clearer (with this clearly explained in the README as well).

Reply:
Makes sense. I thought DAEMON_ is a pattern for all Cosmovisor env variables, but if not, what you are saying indeed makes sense. I'll update it.

Reply:
Fixed.
For README, I thought I commited it already but never did somehow. Should be included now as well.

Reply:
Perfect, yes `DAEMON` is a pattern for action on the app, but it could have been confusing with it disabling the application logs.